src/ApiCompat/ApiCompatRunner.cs

```diff
@@ -34,11 +34,12 @@ public ExportCciSettings()
         public IEqualityComparer<ITypeReference> Settings { get; set; }
     }
 
-    public class Program
+    public static class ApiCompatRunner
     {
-        public static int Main(string[] args)
+        public static int ValidateApiCompat(string args)
         {
-            ParseCommandLine(args);
+            string[] argsArray = args.Split(' ');
```

Comment:
I suspect this will actually break if there are any quoted paths with a space in them.